payments-core/src/main/java/com/stripe/android/googlepay/GooglePayResult.kt

```diff
@@ -0,0 +1,70 @@
+package com.stripe.android.googlepay
+
+import android.content.Intent
+import android.os.Bundle
+import android.os.Parcel
+import androidx.core.os.bundleOf
+import com.google.android.gms.common.api.Status
+import com.stripe.android.model.PaymentMethod
+import com.stripe.android.model.ShippingInformation
+import com.stripe.android.view.ActivityStarter
+import kotlinx.parcelize.Parceler
+import kotlinx.parcelize.Parcelize
+
+internal sealed class GooglePayResult : ActivityStarter.Result {
```

Comment:
This newly added class name collides with `com.stripe.android.model.GooglePayResult`, should we rename it?

Reply:
I'll rename this class to `GooglePaySheetResult`